DockerHubBackend/DockerHubBackend/Data/DatabaseContextSeed.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DockerHubBackend.Models;
+using Microsoft.AspNetCore.Identity;
+
+namespace DockerHubBackend.Data
+{
+    public static class DatabaseContextSeed
```

Comment:
What is this Class used for? Is this for the superuser? Why are two users saved?

Reply:
It's just a script to prepopulate database for development purposes.